src/components/Pricing/data.ts

```diff
@@ -0,0 +1,499 @@
+    },
+    enterpriseExecutors: {
+        label: '4 executors',
+        description:
+            'Offload expensive tasks when running Batch Changes server-side or using code navigation’s auto-indexing functionality.',
+    },
+
+    ...DEPLOYMENT_FEATURE_INFO,
+
+    // Support
+    slaSupport: {
+        label: 'Priority support SLAs',
+        description: 'Priority ticket handling and guaranteed initial response SLA from a dedicated team.',
+    },
+    dedicatedManager: {
+        label: 'Dedicated Technical Account Manager',
+        description:
+            'Dedicated technical account manager to support usage, training, enablement, technical strategy, and overall health.',
+    },
+}
+
+export const ALL_FEATURE_INFO: Record<string, FeatureInfo> = {
+    ...CIP_FEATURE_INFO,
+
+    // Code host integrations
+    codeHost: {
+        label: 'Code host integrations',
+        description: 'Search, understand, fix, and automate across multiple code hosts.',
+    },
+    cloudHosts: {
+        label: 'Cloud code hosts',
+        description: 'Integrate with GitHub.com, GitLab.com, and Bitbucket Cloud.',
+    },
+    selfHosts: {
+        label: 'Self-hosted code hosts',
+        description:
+            'Integrate with Github Enterprise Self-hosted, GitLab Self-hosted, Bitbucket Server/Data Center, and Perforce.',
+    },
+    enterpriseHosts: {
+        label: 'Enterprise-only code hosts',
+        description: 'Integrate with any Git-based code host via src-srv-git.',
+    },
+    privateHosts: {
+        label: 'Private code hosts',
+        description: 'Integrate with supported code hosts stored behind a firewall.',
+    },
+
+    // Security and admin
+    soc2: {
+        label: 'SOC 2 Type II',
+        description: '',
+    },
+    ...SECURITY_FEATURE_INFO,
+
+    // Scale and performance
+    codeStorag: {
+        label: 'Cloud code storage',
+        description: 'Included storage for Cloud deployments.',
+    },
+    executors: {
+        label: 'Executors',
+        description:
+            'Offload expensive tasks when running Batch Changes server-side or using code navigation’s auto-indexing functionality.',
+    },
+
+    // Support
+    support: {
+        label: 'Support',
+    },
+    dedicatedManager: {
+        label: 'Dedicated Technical Account Manager',
+        description:
+            'Dedicated technical account manager to support usage, training, enablement, technical strategy, and overall health.',
+    },
+    supportSla: {
+        label: 'Support SLA',
+        description: 'Priority ticket handling and guaranteed initial response SLA from a dedicated team.',
+    },
+
+    // Deployment
+    ...DEPLOYMENT_FEATURE_INFO,
+    airGappedDeploy: {
+        label: 'Air-gapped deployment',
+        description: 'Deploy Sourcegraph in an air-gapped environment on your own infrastructure (add-on).',
```

Comment:
The ` (add-on)` feels like it belongs outside of the sentence's period because it describes a characteristic of the feature (it's an add-on feature), it's not a parenthetical to the sentence.